### ISSUE TITLE
Move jobs to jobs folder

### ISSUE THREAD

**Issue by conbrad** [bcgov/wps#1601]
**Describe the task**
Move cronjobs to jobs folder, so we know where to look for jobs.

**Acceptance Criteria**
- [ ] api/app/weather_models/env_canada.py -> api/app/jobs/env_canada.py
- [ ] api/app/c_haines/worker.py -> api/app/jobs/worker.py (c_haines_worker.py?)

**Additional context**
- Add any other context about the task here.
- Or here

**Comment by conbrad** [comment]
Closing as this is already in place.

**Comment by Sybrand** [comment]
@conbrad it's on in place for hourly actuals and noons - not for models or c-haines.

**Comment by conbrad** [comment]
> @conbrad it's not in place for hourly actuals and noons - not for models or c-haines. re-opening.

Oh, looks like hourly actuals and noons are here though?
https://github.com/bcgov/wps/tree/main/api/app/jobs

**Comment by Sybrand** [comment]
Yeah - sorry - fat fingers. In for hour and noons, but not for everything else.
